### DISCUSSION TITLE
Cmd-tab to ghostty causing issues with nvim/vim trackpad scrolling

### DISCUSSION THREAD

**Discussion by nighthawk469** [ghostty-org/ghostty#3115] [Feature Requests, Ideas] [answered]
With nvim open in ghostty, if i go to a different macOS application, then Cmd-Tab to return to ghostty, when I try to trackpad scroll in nvim, no scrolling happens. 

**To get scrolling to work, I need to move the mouse cursor any amount**, only then does nvim recognize my trackpad scrolling.

Local macOS nvim (nvim 0.10.3) has this problem, vim locally has no problem.

Over ssh, both nvim and vim have this scrolling issue after using Cmd-Tab.

Clicking the ghostty Dock icon gives no issues regarding trackpad scrolling, because there will be a lot of mouse movement.

Terminal.app has no issues with Cmd-Tab and scrolling.

**Comment by mitchellh** [comment]
Hm. I tried to reproduce this and can’t (I use nvim as my primary editor). Is this plugin related? Is anyone else able to reproduce this?

**Reply by mitchellh** [reply, marked as answer]
Oh sorry! I got it! #3117. Sorry about that.

**Reply by mitchellh** [reply]
Resolved in #3133
